apps/web/package.json

```diff
@@ -43,6 +43,7 @@
     "@heroicons/react": "^1.0.6",
     "@hookform/error-message": "^2.0.0",
     "@hookform/resolvers": "^2.9.7",
+    "@lexical/react": "^0.5.0",
```

Comment:
I thin these belong to `packages/features/package.json`

Reply:
I moved them to [`packages/features/package.json`](https://github.com/calcom/cal.com/pull/5654/commits/9088a63e8867db8818545c849e343829d2b68600)